test/ctr.bats

```diff
@@ -387,7 +387,6 @@ function teardown() {
 	run ocic ctr execsync --id "$ctr_id" doesnotexist
 	echo "$output"
 	[ "$status" -ne 0 ]
-	[[ "$output" =~ "executable file not found in" ]]
```

Comment:
This string was returned by runC itself. Other runtimes may not decide to return the same string.
I think the only thing we can check here is that this execsync really failed.